components/kyma-environment-broker/internal/storage/storage_test.go

```diff
@@ -142,6 +144,11 @@ func TestSchemaInitializer(t *testing.T) {
 		err = svc.InsertProvisioningOperation(givenOperation)
 		require.NoError(t, err)
 
+		ops, err := svc.GetOperationsInProgressByType(dbsession.OperationTypeProvision)
+		require.NoError(t, err)
+		assert.Len(t, ops, 1)
+		assert.Equal(t, givenOperation.Operation, ops[0])
```

Comment:
there is an assertProvisioningOperation function

Reply:
please, run the test and fix the build tag and makefile

Reply:
👍

Reply:
about `assertProvisioningOperation`, I created a similar function `assertOperation`, which compares `internal.Operation` structs.